content/07.prior-knowledge.md

```diff
@@ -26,8 +26,8 @@ Another approach that builds on prior knowledge and large volumes of related dat
 Transfer learning leverages shared features, e.g., normal developmental processes that are aberrant in disease or an imaging anomaly present in both rare and common diseases, to advance our understanding of rare diseases. 
 Transfer learning, where a model trained for one task or domain (source domain) is applied to another related task or domain (target domain), can be supervised or unsupervised. 
 Among various types of transfer learning, feature-representation-transfer approaches learn representations from the source domain and apply them to a target domain [@doi:10.1109/tkde.2009.191](Figure [@fig:4]c-e).
-That is, representation learning, as discussed in an earlier section, does not need to be applied only to describe the dataset on which the algorithm was trained – it can also be used to elucidate signals in sufficiently similar data.
+That is, representation learning, as discussed in an earlier section, does not need to be applied only to describe the dataset on which the algorithm was trained (Figure [@fig:4]c) – it can also be used to elucidate signals in sufficiently similar data (Figure [@fig:4]e) and may offer an improvement in descriptive capability over models trained on small rare disease datasets alone (Fig [@fig:4]d).
 For instance, low-dimensional representations can be learned from tumor transcriptomic data and transferred to describe patterns associated with genetic alterations in cell line data [@doi:10.1186/s13059-020-02021-3](Figure [@fig:4]e).
 In the next section, we will summarize specific instances of applying transfer learning, along with other techniques described herein, to the study of rare diseases.
 
-![Strategies that build upon prior knowledge help ML models learn patterns in rare disease datasets. A) Knowledge graphs integrate different data types and may allow models to learn from connections that are rare disease-specific or happen in many biomedical contexts. B) Transfer learning is when a model trained in for one task or domain is applied to another, related task.](images/figures/pdfs/figure-4-KG-transfer-learning.png){#fig:4}
+![Strategies that build upon prior knowledge help ML models learn patterns in rare disease datasets. A) Knowledge graphs integrate different data types and may allow models to learn from connections that are rare disease-specific or happen in many biomedical contexts. B) TODO: Describe Li et al. (?) panel C) A model can learn a representation from a collection of samples that do not directly assay a rare disease of interest. D) Representation learning models trained on rare disease datasets may learn representations that are less useful for descriptive analysis of those datasets due to the sample size and lack of variation. E) Learned representations from sufficiently related data (from panel C) can be transferred to rare disease dataset (from panel D) for improved descriptive analyses.](images/figures/pdfs/figure-4-KG-transfer-learning.png){#fig:4}
```

Comment:
Panel B needs to be described. We might want to alter the description panel A as well, not sure. cc: @allaway

Reply:
```suggestion
![Strategies that build upon prior knowledge help ML models learn patterns in rare disease datasets. A) Knowledge graphs integrate different data types and may allow models to learn from connections that are rare disease-specific or happen in many biomedical contexts. B) Knowledge graphs can also be used to augment data. Li et. al.[@doi:10.1186/s12911-019-0938-1] applied a classifier to an EHR corpus to identify rare disease patients. They trained a classifier on the EHR data alone (e.g., thrombocytopenia, anemia) and trained another classifier on data augmented with medically-related concepts from a knowledge graph (e.g., neutropenia, stroke). The classifier trained on knowledge-graph augmented data has lower error and higher accuracy.  C) A model can learn a representation from a collection of samples that do not directly assay a rare disease of interest. D) Representation learning models trained on rare disease datasets may learn representations that are less useful for descriptive analysis of those datasets due to the sample size and lack of variation. E) Learned representations from sufficiently related data (from panel C) can be transferred to rare disease dataset (from panel D) for improved descriptive analyses.](images/figures/pdfs/figure-4-KG-transfer-learning.png){#fig:4}
```